### ISSUE TITLE
[BITV] Preview card in personal settings might not satisfy contrast requirements

### ISSUE THREAD

**Issue by susnux** [nextcloud/server#42440]
The preview card in the personal settings uses `primary-element` as background color but overlays it with a `primary-gradient` which is using `primary-color` and then uses `primary-element-text` for the username.
But this combination is not always matching meaning either `primary-element-*` **or** `primary` should be used. See example where black is used but does not provide enough contrast:

![Screenshot 2023-12-22 at 01-35-12 Persönliche Informationen - Persönliche Einstellungen - Nextcloud](https://github.com/nextcloud/server/assets/1855448/a4e620b8-9318-42de-a878-773b2290cd20)

**Comment by emoral435** [comment]
Will be closed and needed to be checked once #42622 gets merged 🙂

**Comment by emoral435** [comment]
Merged!
